os x/assemble_app.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# The qt bin folder needs to be in your PATH for this script to work.
+rm -rf QJackTrip.app
+[ ! -d "QJackTrip.app_template/Contents/MacOS" ] && mkdir QJackTrip.app_template/Contents/MacOS
+cp -a QJackTrip.app_template QJackTrip.app
+cp -f ../builddir/qjacktrip QJackTrip.app/Contents/MacOS/
+
+# If you want to create a signed package, uncomment and modify the codesign parameter below as appropriate.
+macdeployqt QJackTrip.app #-codesign="Developer ID Application: Aaron Wyatt"
+exit 0
+# Needed for notarization:
+# Remove the exit line above and edit the signing identity below as appropriate.
+mkdir -p QJackTrip.app/Contents/Frameworks/Jackmp.framework/Versions/A/Resources
```

Comment:
Ahh, this is totally out of date, and was a kludge that I needed to use to get the old, old version of Jack running with a bundled version of QJackTrip. It's no longer a thing and not in the version of the script that I personally use (which also has all of my developer account credentials in it for signing, so have to strip those from it first before updating here). My version also builds an installer using Packages, and then automatically submits the package to apple for notarization. (It could probably be done without Packages, using just the apple dev tools and a whole lot of scripting, but I haven't had the time to look into that.) Also, I've taken the approach of having the installer create a symlink in /usr/local/bin and feel that that's probably more idiomatic, but that may just by my linux background talking.

Reply:
I see, thanks.
Would you be willing to add docs for using these scripts, i.e. how to make an app bundle? Then I'd be happy to contribute and add that to the GHA.

Reply:
Yes, I can do that. In the meantime, if you just run the assemble_app.sh script it should create a working app bundle. (It just won't create the installer package or sign anything without tweaking.)